src/ecs/commands/runCommandInContainer.ts

```diff
@@ -121,11 +118,24 @@ export async function runCommandInContainer(
         if (!command) {
             return
         }
+        let ssmPlugin: string | undefined
+        if (!settings.readDevSetting<boolean>('aws.dev.forceInstallTools', 'boolean', true)) {
+            ssmPlugin = await getCliCommand(AWS_CLIS['session-manager-plugin']!)
+        }
+
+        if (!ssmPlugin) {
+            ssmPlugin = await installCli('session-manager-plugin', true)
```

Comment:
This can throw. We probably should just narrow the valid types to what we support, not what the telemetry can report.